Modules/_sqlite/cursor.c

```diff
@@ -278,9 +278,14 @@ _pysqlite_fetch_one_row(pysqlite_Cursor* self)
             converter = Py_None;
         }
 
+        /*
+         * Note, sqlite3_column_bytes() must come after sqlite3_column_blob()
+         * or sqlite3_column_text().
+         * Ref. https://sqlite.org/c3ref/column_blob.html
```

Comment:
Nitpick:  "See XXX" or "See XXX for more (information|details)" are more common than "Ref." in the codebase. I'd prefer to stay consistent with the existing conventions.

Reply:
Noted, thanks! Consistency FTW. 

(I use "ref." when I write Norwegian. It tends to sneak in when I write English.)

Reply:
See d6be86f